api/src/pcapi/notifications/push/backends/testing.py

```diff
@@ -27,3 +27,6 @@ def send_transactional_notification(self, notification_data: TransactionalNotifi
     def delete_user_attributes(self, user_id: int) -> None:
         super().delete_user_attributes(user_id)
         testing.requests.append({"user_id": user_id})
+
+    def send_transactional_notification_delayed(self, notification_data: TransactionalNotificationData) -> None:
+        self.send_transactional_notification(notification_data)
```

Comment:
(style / question de goût) idem plus haut

```suggestion
    send_transactional_notification_delayed = send_transactional_notification
```